### PR TITLE
XAMARIN-41237 : Prepare UG content for show leading view and show trailing view support.

### PR DESCRIPTION
**Pull request by BalachandarSelvam**
Hi Team,

To include UG content for ShowLeadingView and ShowTrailingView in TextInputLayout.

Regards,
Balachandar.

### CODE REVIEW

**Comment by BalachandarSelvam** [comment]
rebuild

File: Xamarin/Text-Input-Layout/Custom-Icons.md [hunk 1]
```diff
@@ -142,6 +142,53 @@ inputLayout.InputView = new Entry();
 
 ![customisation](Custom-Icons-images/textInput_icons_img5.png)
 
+## Leading and trailing view visibility customization
+
+The ShowLeadingView and ShowTrailingView properties in SfTextInputLayout can be used to control the visibility of the leading and trailing views. Even if the content is already loaded in their respective views, their visibility can be changed.
```
Comment (vigneshrameshsync) [inline Xamarin/Text-Input-Layout/Custom-Icons.md:147]: The [`ShowLeadingView`]() and [`ShowTrailingView`]() properties in [`SfTextInputLayout`](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.TextInputLayout.SfTextInputLayout.html) can be used to control the visibility of the leading and trailing views respectively.

File: Xamarin/Text-Input-Layout/Custom-Icons.md [hunk 2]
```diff
@@ -142,6 +142,53 @@ inputLayout.InputView = new Entry();
...
+
+{% tabs %}
+
+{% highlight xaml %} 
+
+  <inputLayout:SfTextInputLayout
+    Hint="Birth date"
+    LeadingViewPosition="Inside"
```
Comment (vigneshrameshsync) [inline Xamarin/Text-Input-Layout/Custom-Icons.md:155]: Remove unnecessary code lines.

File: Xamarin/Text-Input-Layout/Custom-Icons.md [hunk 3]
```diff
@@ -142,6 +142,53 @@ inputLayout.InputView = new Entry();
...
+inputLayout.InputView = new Entry();
+inputLayout.ContainerType = ContainerType.Outlined;
+inputLayout.ShowLeadingView = false;
+inputLayout.ShowTrailingView = false;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Visibility customisation](Custom-Icons-images/textInput_icons_img6.gif)
```
Comment (vigneshrameshsync) [inline Xamarin/Text-Input-Layout/Custom-Icons.md:190]: Provide proper file name.

**Review by vigneshrameshsync — CHANGES_REQUESTED** [review]

File: Xamarin/Text-Input-Layout/Custom-Icons.md (hunk 1 above)
Comment (BalachandarSelvam) [reply Xamarin/Text-Input-Layout/Custom-Icons.md]: Simplified the content from from text.

File: Xamarin/Text-Input-Layout/Custom-Icons.md (hunk 2 above)
Comment (BalachandarSelvam) [reply Xamarin/Text-Input-Layout/Custom-Icons.md]: Removed LeadingViewPosition.

File: Xamarin/Text-Input-Layout/Custom-Icons.md (hunk 3 above)
Comment (BalachandarSelvam) [reply Xamarin/Text-Input-Layout/Custom-Icons.md]: Provided proper file name.

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

**Pull request merged**